fda/basis.py

```diff
@@ -827,6 +876,8 @@ def __init__(self, domain_range=(0, 1), nbasis=3, period=1):
         nbasis += 1 - nbasis % 2
         super().__init__(domain_range, nbasis)
 
+        self.default_extrapolation = Extrapolation.extrapolation
```

Comment:
I would call it `extrapolation` rather than `default_extrapolation`. `default_extrapolation` sounds like a property of the **class**, more than a property of the object.

Reply:
Changed in 85c76d1, but could be a good idea change it as a property of the class of the basis.

Reply:
Added to #12